File: Web_search_job/ClientApp/src/app/models/backend/dtos/jobs/job-request.dto.ts
import {ResumeShortDTO} from "../profiles/resume-short.dto";

export interface JobRequestDTO {
  id: number;
  jobId: number;
  userId: number;
  resume?: ResumeShortDTO;
  resumeURL?: string;
  coverLetter?: string;
  positives?: string;
  projects?: string;
  status?: string;
  createdAt: Date;
}

File: Web_search_job/DTO/Job/JobRequestDTO.cs
using Web_search_job.DTO.Resume;

namespace Web_search_job.DTO.Job
{
    public class JobRequestDTO
    {
        public int? Id { get; set; }
        public int JobId { get; set; }
        public int UserId { get; set; }
        public ResumeShortDTO? Resume { get; set; }
        public string? ResumeURL { get; set; }
        public string? CoverLetter { get; set; }
        public string? Positives { get; set; }
        public string? Projects { get; set; }
        public string? Status { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}
